airflow/ui/src/pages/TaskInstance/Logs/TaskLogContent.tsx

```diff
@@ -30,24 +32,47 @@ type Props = {
   readonly wrap: boolean;
 };
 
-export const TaskLogContent = ({ error, isLoading, logError, parsedLogs, wrap }: Props) => (
-  <Box>
-    <ErrorAlert error={error ?? logError} />
-    <ProgressBar size="xs" visibility={isLoading ? "visible" : "hidden"} />
-    <Code
-      css={{
-        "& *::selection": {
-          bg: "blue.subtle",
-        },
-      }}
-      overflow="auto"
-      py={3}
-      textWrap={wrap ? "pre" : "nowrap"}
-      width="100%"
-    >
-      <VStack alignItems="flex-start" gap={0}>
-        {parsedLogs}
-      </VStack>
-    </Code>
-  </Box>
-);
+export const TaskLogContent = ({ error, isLoading, logError, parsedLogs, wrap }: Props) => {
+  const { colorMode } = useColorMode();
+  const [bgLight, bgDark] = useToken("colors", ["blue.400", "blue.700"]);
```

Comment:
Why a custom color instead of `blue.emphasized`?

Reply:
Thanks, updated. I was trying out `bg.emphasized` and didn't know each color had variants.

https://www.chakra-ui.com/docs/theming/colors#semantic-tokens